source/install-desktops/enable-reverse-proxy.rst

```diff
@@ -0,0 +1,149 @@
+.. _install-desktops-enable-reverse-proxy:
+
+Enable Reverse Proxy
+====================
+
+The reverse proxy will proxy a request to any specified host and port through
+IP sockets. This is different than what is used for proxying to per-user NGINX
+processes through Unix domain sockets. This can be used to connect to Jupyter
+notebook servers, RStudio servers, VNC servers, and more... This is disabled by
+default as it can be a security risk if not properly setup using a good
+``host_regex``.
+
+Requirements:
+
+- a regular expression that best describes all the hosts that you would want a
+  user to connect to through the proxy (e.g., ``[\w.-]+\.osc\.edu``)
+- confirm that if you run the command ``hostname`` from a compute node it will
+  return a string that matches the above regular expression
+
+  .. code-block:: sh
+
+     $ hostname
+     n0001.ten.osc.edu
+
+  .. note::
+
+     If the :command:`hostname` command gives you a value that cannot be used
+     to connect to the compute node from the OnDemand host, then you can
+     override it in the cluster config with a Bash command that will work,
+     e.g.:
+
+     .. code-block:: yaml
+
+        # /etc/ood/config/clusters.d/cluster1.yml
+        ---
+        v2:
+          # ...
+          # ... other configuration options ...
+          # ...
+          batch_connect:
+            basic:
+              # ...
+              # set_host: "host=$(hostname)"
+              set_host: "host=$(hostname -A | awk '{print $1}')"
+            vnc:
+              # ...
+              # set_host: "host=$(hostname)"
+              set_host: "host=$(hostname -A | awk '{print $1}')"
+
+#. We will update the Apache configuration file by adding ``Location``
```

Comment:
above this, add a small header "Steps to enable reverse proxy:" because we are now in a new section after "Requirements:"